src/contrib/cluster/Akka.Cluster.Tools/PublishSubscribe/OverflowStrategy.cs

```diff
@@ -0,0 +1,39 @@
+﻿// -----------------------------------------------------------------------
+//  <copyright file="OverflowStrategy.cs" company="Akka.NET Project">
+//      Copyright (C) 2009-2022 Lightbend Inc. <http://www.lightbend.com>
+//      Copyright (C) 2013-2025 .NET Foundation <https://github.com/akkadotnet/akka.net>
+//  </copyright>
+// -----------------------------------------------------------------------
+
+namespace Akka.Cluster.Tools.PublishSubscribe;
+
+/// <summary>
+/// Represents a strategy that decides how to deal with a buffer that is full but is about to receive a new message.
+/// </summary>
+public enum OverflowStrategy
```

Comment:
Make the buffer circular and just fail older messages with a NACK.

Reply:
Again, if users want something fancier - they should be setting up RabbitMQ instances